<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #121212;
    color: white;
    font-family: Verdana;
    font-size: 16pt;
    margin: 10px;
}

QPushButton {
    border: 2px solid gray;
    border-radius: 5px;
	border-color: white;
}

QPushButton#btn_add, #btn_save, #btn_gen_password, #btn_idk {
    padding: 3px;
}

QPushButton:hover {
    border-color: #090;
}

QPushButton:pressed {
    border: 4px solid #090;
    border-radius: 5px;
}

QPushButton:checked {
	background-color: #006300;
	border-color: #090;
}

QTableWidget {
    background-color: #1e1e1e;
    color: #f0f0f0;
    gridline-color: #3a3a3a;
    font-size: 14px;
}

QHeaderView::section {
    background-color: #2c2c2c;
    color: #f0f0f0;
    font-weight: bold;
    padding: 4px;
    border: 1px solid #3a3a3a;
}

QTableWidget::item:selected {
    background-color: #3a76f0;
    color: white;
}

QTableWidget {
    alternate-background-color: #2a2a2a;
}
QTableCornerButton::section {
    background-color: #2c2c2c;   /* фон кнопки */
    border: 1px solid #3a3a3a;   /* рамка */
}

QTableCornerButton::section:hover {
    background-color: #3a76f0;   /* при наведении */
}

QTableCornerButton::section:pressed {
    background-color: #2a5bc0;   /* при нажатии */
}


QLineEdit {
    background-color: #111;           /* тёмный фон */
    color: #fff;                      /* текст пользователя */
    border: 2px solid #444;           /* обычная рамка */
    border-radius: 8px;
    padding: 6px 10px;
    font-size: 14px;
    selection-background-color: #555; /* выделение текста */
}

QLineEdit:focus {
    border: 2px solid #1E90FF;       /* рамка при фокусе */
}

QLineEdit::placeholder {
    color: #888;                      /* светло-серый текст плейсхолдера */
    font-style: italic;
}
QLineEdit {
    background-color: #111;           /* тёмный фон */
    color: #fff;                      /* текст пользователя */
    border: 2px solid #444;           /* обычная рамка */
    border-radius: 8px;
    padding: 6px 10px;
    font-size: 14px;
    selection-background-color: #555; /* выделение текста */
}

QLineEdit:focus {
    border: 2px solid #1E90FF;       /* рамка при фокусе */
}

QLineEdit::placeholder {
    color: #888;                      /* светло-серый текст плейсхолдера */
    font-style: italic;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="input_site">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>service</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_login">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_password">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="text">
         <string>DELETE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_back">
        <property name="text">
         <string>BACK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::ContiguousSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
